bootstrap/standalone/src/main/resources/permissions.yml

```diff
@@ -0,0 +1,9 @@
+
+# Add any permissions here that all connections should have.
```

Comment:
I would call it "players" instead of connections - this is exposed to the user, and players is more straightforward